docs/sources/alerting/difference-old-new.md

```diff
@@ -1,18 +1,15 @@
 +++
-title = " Difference between unified and old Grafana alerts"
-description = "Pause an existing alert rule"
-keywords = ["grafana", "alerting", "guide", "rules", "view"]
+title = "What's New with Unified Alerts"
+description = "What's New with Unified Alerts"
+keywords = ["grafana", "alerting", "guide"]
 weight = 112
 +++
 
-# Difference between unified and old Grafana alerts
+# Differences between Unified Alerts and Grafana dashboard alerts
+The Unified Alerts released with Grafana 8.0 as an opt-in feature centralizes alerting information for Grafana managed alerts and alerts from Prometheus-compatible datasources in one UI and API. You are able to create and edit alerting rules for Grafana managed alerts and see alerting information from Prometheus-compatible datasources in a single, searchable view.
 
-The `ngalert` feature toggle enables the beta version of our new alerting system. 
+## Create alerts outside of Dashboards
+Grafana dashboard alerts were exactly that: alerts that were tied to a dashboard. Unified Alerts allows you to create queries and expressions that can combine data from multiple sources, in unique ways. You are still able to link dashboards and panels to alerts, allowing you to quickly troubleshoot the system under observation, by linking a dashboard and/or panel ID to the alerting rule. 
 
->**Note:** It is recommended to backup Grafana's database before enabling this feature.
-
-When the feature flag is enabled, dashboard alerting is disabled and dashboard alerts are migrated into the system. Going to "Alert List" will take you to the new system.
-
-Once you disable the new alters, all migrated and newly created alerts in the new system are deleted, and dashboard alerting will be enabled again.
-
-During beta, the migration of existing dashboard rules may change.
\ No newline at end of file
+## View and search for alerts from Prometheus
```

Comment:
also create and edit alerts from Cortex or Loki!

Reply:
also maybe worth mentioning possibility to manage cortex alertmanager & grafana multidimensional alerts?

Reply:
Good callouts, @domasx2 Added some stuff for that